pkg/manager/impl/launch_plan_manager.go

```diff
@@ -142,6 +142,9 @@ func isScheduleEmpty(launchPlanSpec admin.LaunchPlanSpec) bool {
 	if schedule == nil {
 		return true
 	}
+	if schedule.GetCronSchedule() != nil && len(schedule.GetCronSchedule().Schedule) != 0 {
+		return false
+	}
 	if len(schedule.GetKickoffTimeInputArg()) != 0 {
 		return false
 	}
```

Comment:
@katrogan i didn't change this previous logic but trying to understand why do we have this check . We can't have a schedule with just this arg so wanted to understand the purpose of this check

Reply:
great question (and git blame is not very helpful here) I think we can safely assume the schedule expression oneof is not nil then that's all we need to check

Reply:
Ok cool.Merging this will remove that logic in followup